emepuchades/task-manager
# feat(commitizen setup config): install commitizen deps to format comm…

…it messages

# Task Manager Pull Request 📅

## Descripcion ✏️
Configuration of Commitizen to give proper format to commit messages

## Visual Changes 🎨
My PR doesn't include visual changes. I made sure to run an npm install to see if the application loads correctly in the browser after this installation

## How to test 🐛
To make sure your commit message has the proper format, please write git cz, instead of git commit as usual and you'll be prompted to fill in the required fields.
